lib/searchable_by.rb

```diff
@@ -44,7 +44,8 @@ def scope(&block)
     def self.norm_values(query)
       values = []
       query  = query.to_s.dup
-      query.gsub!(/(\-|\+?)"([^"]+)"/) {|_| values.push("#{Regexp.last_match(1)}#{Regexp.last_match(2)}"); '' }
+      query.gsub!(/([\-\+]?)"+([^"]*)"+/) {|_| values.push("#{Regexp.last_match(1)}#{Regexp.last_match(2)}"); '' }
+
       values.concat query.split(' ')
       values.uniq!
```

Comment:
@sashadw sorry, looking at the specs: we do still need `values.reject!(&:blank?)` (before uniq!)